src/lib/dateRangeHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import { DateTime } from 'luxon';
+
+export const getTwelveMonthReportDateRange = (): {
+  startDate: string;
+  endDate: string;
+} => {
+  const startDate = DateTime.now()
+    .startOf('month')
+    .minus({ months: 12 })
+    .toISODate();
+  const endDate = DateTime.now()
+    .startOf('month')
+    .minus({ days: 1 })
+    .endOf('day')
```

Comment:
Do you think that `.minus({ months: 1 }).endOf('month')` would be clearer? Since what we want is the end of the previous month.